src/main/kotlin/org/rescado/server/service/GroupService.kt

```diff
@@ -64,11 +77,13 @@ class GroupService(
         return membershipRepository.save(membership)
     }
 
-    fun deleteMembership(membership: Membership) = membershipRepository.delete(membership)
+    fun deleteMembership(membership: Membership) {
+        membershipRepository.delete(membership)
 
-    // TODO implement
-// @Scheduled(cron = daily?)
-// fun cleanup(){
-//     remove groups that have no members and were created >1 month ago
-// }
+        // If after leaving, the group has no confirmed members anymore
+        if (membership.group.memberships.any { it.status == Membership.Status.CONFIRMED }) {
```

Comment:
Moet hier geen `!` ? 

of `if (membership.group.memberships.none { it.status == Membership.Status.CONFIRMED }) `

Reply:
👍